### ISSUE TITLE
"Duplicate class found" build error after upgrading to 3.29.0

### ISSUE THREAD

**Issue by zigavehovec** [googleads/googleads-ima-android#111]
Hello!

Since we updated IMA SDK’s version from `3.26.0` to `3.29.0` we have been getting the following build error: 

```
Duplicate class androidx.lifecycle.ViewModelLazy found in modules lifecycle-viewmodel-2.5.1-runtime (androidx.lifecycle:lifecycle-viewmodel:2.5.1) and lifecycle-viewmodel-ktx-2.3.1-runtime (androidx.lifecycle:lifecycle-viewmodel-ktx:2.3.1)
Duplicate class androidx.lifecycle.ViewTreeViewModelKt found in modules lifecycle-viewmodel-2.5.1-runtime (androidx.lifecycle:lifecycle-viewmodel:2.5.1) and lifecycle-viewmodel-ktx-2.3.1-runtime (androidx.lifecycle:lifecycle-viewmodel-ktx:2.3.1)
```

We looked into the dependency tree and figured out that the IMA SDK’s transitive dependencies don't work well with the latest appcompat version (1.5.1). 

**To replicate the issue:** 
- Change `androidx.appcompat:appcompat` version in `basicexample/app/build.gradle` to `1.5.1`
- Build the project

**Note that:**
- IMA SDK version `3.29.0` works fine with `androidx.appcompat:appcompat:1.4.1`
- IMA SDK version `3.26.0` works fine with `androidx.appcompat:appcompat:1.5.1`

Adding this for others that might have the same problem - we are currently using this workaround to be able to keep the newest appcompat version: 

```
// IMA SDK depends on lifecycle-viewmodel-ktx, which if resolved to version lower than 2.5.1
// duplicates some classes from appcompat v1.5.1 and thus breaks the build.
implementation('androidx.lifecycle:lifecycle-viewmodel-ktx') {
    version {
        require '2.5.1'
    }
}
```

**Comment by Kiro705** [comment]
Hello @zigavehovec ,

Thank you for bringing up this issue.

With the new release of `androidx.appcompat:appcompat:1.6.0`, I think this resolves the issue. Can you confirm if this is still happening in your apps with the 1.6.0 version?

Thank you,
Jackson
IMA SDK team

**Comment by zigavehovec** [comment]
Hi @Kiro705!

Thank you for getting back.
I can confirm that the issue has been fixed with the latest appcompat version (`1.6.1`). 

Greetings, Žiga
